include/ipmitool/ipmi_fru.h

```diff
@@ -503,11 +503,11 @@ struct fru_picmgext_amc_p2p_record {
 	unsigned char resource_id         :4;
 	unsigned char /* reserved */      :3;
 	unsigned char record_type         :1;
-#else	
+#else
 	unsigned char record_type         :1;
 	unsigned char /* reserved */      :3;
 	unsigned char resource_id         :4;
-#endif 
+#endif
```

Comment:
Just a side note. We'll have to get rid of bit-fields in future. There is #59 for that.